### ISSUE TITLE
Link blue needs higher contrast

### ISSUE THREAD

**Issue by elisabethirgens** [bring/hedwig#100]
The [current link blue](https://hedwig-docs.herokuapp.com/bring.html#/Colors) has lower contrast than it should have. 

It will fail accessibility requirements when used on the default light gray background `#f2f2f2`. (Also when used on white background, the contrast ratio is very low at 3.3 and a matter of interpretation around font-size, font-weight and font-family…)

| Link blue | Contrast Ratio | Accessible | Hue | Saturation | Lightness |
| :---: | :---: | :---: |  ---: |  ---: |  ---: |
| `#4a90e2`  |  2.9   |  🚫  |  212° | 72% |  59% |  
| `#0066dd`  | 4.8   |  ✅  |  212° | 100% |  43% |  

We can keep the same hue, but increase saturation and decrease lightness.

**Suggestion:** Change link blue to [pass WCAG 2.0 contrast requirements](http://leaverou.github.io/contrast-ratio/#%230066dd-on-%23f2f2f2). ✌️
